projects/dsp-ui/src/lib/viewer/viewer.module.ts

```diff
@@ -116,7 +120,8 @@ import { AddValueComponent } from './operations/add-value/add-value.component';
         JDNDatepickerDirective,
         ResourceListComponent,
         ResourceGridComponent,
-        AddValueComponent
+        AddValueComponent,
+		StillImageComponent
```

Comment:
There's some weird spacing here, please align :)

Reply:
done in https://github.com/dasch-swiss/dsp-ui-lib/pull/128/commits/f9eaf3a1367774f5994276320a7f81af490e3578